app/helpers/activity_helper.rb

```diff
@@ -4,10 +4,17 @@ def step_is_complete_or_next?(activity:, step:)
 
     return false if activity.form_state.nil?
     return true if activity.form_steps_completed?
+    return true if activity.fund? && step == :identifier
 
     presenter_position = steps.index(step.to_sym)
     activity_position = steps.index(activity.form_state.to_sym)
 
     presenter_position <= activity_position + 1
   end
+
+  def link_to_activity_parent(parent:, user:)
```

Comment:
I think this is a great improvement. You mentioned a concern about the `current_user` but I wasn't entirely clear on the worry. If it's around passing in user: when `current_user` is technicall available then I think being explicit about the dependencies of this method is preferable. 

It makes the method interface explicit, users know what is required for the method to work, rather than magical calls out to other methods. That in turn allows us more options when testing it. We can inject doubles rather worrying about creating a user object.